src/main.zig

```diff
@@ -1343,7 +1341,7 @@ pub const Editor = struct {
                     }
 
                     if (pollfds[0].revents & SystemCapabilities.POLL_IN != 0) {
-                        self.deferred_action_queue.enqueue(DeferredAction{ .TryUpdateOnce = 0 }) catch {};
+                        self.deferred_action_queue.enqueue(.{ .try_update_once = 0 }) catch {};
```

Comment:
I know you probably don't like the snake case enums but this is the key reason why they ended up like that (historically enums were CamelCase in Zig) - thery're also struct fields in tagged unions.